DOCS/man/mpv.rst

```diff
@@ -299,7 +299,8 @@ g-p
     Select a playlist entry.
 
 g-s
-    Select a subtitle track.
+    Select a subtitle track. In this and other tracks selectors, selecting the
```

Comment:
Maybe put the note above in common selector paragraph. It's weird to have it only in one keybind.